services/skus/credentials.go

```diff
@@ -32,169 +33,162 @@ const (
 	defaultCohort             int16 = 1
 )
 
-// Retry and backoff variables
 var (
-	defaultExpiresAt   = time.Now().Add(17532 * time.Hour) // 2 years
-	retryPolicy        = retrypolicy.DefaultRetry
-	nonRetriableErrors = []int{http.StatusBadRequest, http.StatusUnauthorized, http.StatusForbidden,
-		http.StatusInternalServerError, http.StatusConflict}
-)
-
-var (
-	// ErrOrderUnpaid - unpaid order variable
-	ErrOrderUnpaid = errors.New("order not paid")
-
-	// ErrOrderHasNoItems is an order has no order items error
+	ErrOrderUnpaid     = errors.New("order not paid")
 	ErrOrderHasNoItems = errors.New("order has no items")
-)
 
-// Issuer includes information about a particular credential issuer
-type Issuer struct {
-	ID         uuid.UUID `json:"id" db:"id"`
-	CreatedAt  time.Time `json:"createdAt" db:"created_at"`
-	MerchantID string    `json:"merchantId" db:"merchant_id"`
-	PublicKey  string    `json:"publicKey" db:"public_key"`
-}
+	errInvalidIssuerResp model.Error = "invalid issuer response"
 
-// Name returns the name of the issuer as known by the challenge bypass server
-func (issuer *Issuer) Name() string {
-	return issuer.MerchantID
-}
+	defaultExpiresAt = time.Now().Add(17532 * time.Hour) // 2 years
+	retryPolicy      = retrypolicy.DefaultRetry
+	dontRetryCodes   = map[int]struct{}{
```

Comment:
The old name, `nonRetriableErrors` did not make much sense. Simply because the slice it held did not contain errors; rather – response codes. Additionally, I personally do not like much various `-ables` in code, as they are quite mouthful, and can be rephrased.

So here's my attempt. If it does not work well, am open to suggestions. It should be severely refactored anyway, so I would not spend much time debating it, though.

Reply:
I believe the term non-retriable came from @Sneagan 's work on challenge bypass server attempting to classify errors as retriable/non-retriable.  I agree with you and lets keep it how you have it.